include/libpad.h

```diff
@@ -12,17 +12,6 @@
 
 /* don't change these macros and structures which is referred in controler code */ 
 
-#ifndef TRUE
```

Comment:
Same here. So we may want to keep these TRUE/FALSE/ERROR blocks, or move them to a new header by itself, like sys/return_values.h, and include it from these two files, as the values are basically implicitly needed to evaluate the return values of the functions. Could also probably do with an enum instead, but that's probably too big of a change.